library/std/src/fs.rs

```diff
@@ -2400,17 +2400,17 @@ pub fn create_dir<P: AsRef<Path>>(path: P) -> io::Result<()> {
 ///
 /// # Errors
 ///
-/// This function will return an error in the following situations, but is not
-/// limited to just these cases:
+/// This function will return an error in the following situation, but is not
+/// limited to just this case:
 ///
-/// * If any directory in the path specified by `path`
-/// does not already exist and it could not be created otherwise. The specific
+/// * If any directory specified in `path`
+/// does not already exist and could not be created. The specific
 /// error conditions for when a directory is being created (after it is
 /// determined to not exist) are outlined by [`fs::create_dir`].
```

Comment:
I don't think there is any point in keeping the bulleted list with only one item. Could you reword to something like 

> The function will return an error if any directory specified in `path` does not does not exist and could not be created. There may be other error conditions; see [`fs::create_dir`] for specifics.